engines/bastion_katello/app/assets/javascripts/bastion_katello/common/views/katello-agent-notice.html

```diff
@@ -0,0 +1,8 @@
+<section>
+  <p bst-alert="warning" >
+    <span translate>
+      Katello-agent is deprecated and will be removed in Katello 4.0.  Consider migrating to using Remote Execution.
```

Comment:
nit: is 'migrating to using' correct? I'm actually not sure, but something about it sounds a little strange to my ear. Should it be 'use' or could we remove that word altogether?

Reply:
'migrating to Remote Execution' sounds better, updated